app/controllers/two_factor_authentication/add_methods_controller.rb

```diff
@@ -0,0 +1,43 @@
+module TwoFactorAuthentication
+  class AddMethodsController < ApplicationController
```

Comment:
For this, I'm thinking we may want to stick with more resource-oriented naming. A controller is typically operating (CRUD'ing) on a noun, and representing some 'thing' at its endpoint. (I understand it can be tricky to name at times, depending on what you're doing, and especially with 'method' in the name.)

Reply:
I agree. Controller names should be nouns. Dropping the verb from the name should do it.

Reply:
It needs something to distinguish it from the other controller.  Originally it was 'additional_methods' (a noun) but I shortened it.